source/fundamentals/gridfs.txt

```diff
@@ -87,31 +87,32 @@ Create a GridFS Bucket
 
 To store or retrieve files from GridFS, create a bucket or get a reference to
 an existing bucket on a MongoDB database. To create a ``GridFSBucket`` instance,
-call the ``NewBucket()`` method with a database parameter:
+call the ``GridFSBucket()`` method on a ``Database`` instance, as shown
+in the following code:
 
 .. code-block:: go
 
    db := client.Database("myDB")
-   bucket, err := gridfs.NewBucket(db)
+   bucket, err := db.GridFSBucket()
```

Comment:
This is correct, but may change with https://github.com/mongodb/mongo-go-driver/pull/1533 (which removes the error return value).

Reply:
Thanks for pointing that out, I'll wait to merge until that PR is resolved!